### ISSUE TITLE
[Bug]: date range selector bug

### ISSUE THREAD

**Issue by agaitan026** [lowcoder-org/lowcoder#453]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

when theres is no date selected in date range component in backend i can see a strange date

image attached

![image](https://github.com/lowcoder-org/lowcoder/assets/56446006/b57c6122-156a-47ac-a9a9-74b6cc2f66a5)

### Expected Behavior

It should return NULL of no date or range selected

### Steps to reproduce

create a date range component and check the default values

**Comment by FalkWolsky** [comment]
Agree. We already work on the date components these days and will look to close this bug.

**Comment by placidic** [comment]
The issue stems from the client timezone being +/- GMT time.  As a workaround, you can use dayjs to convert the date value to UTC from local using the formatted date (which includes the offset), and wrap it in a transformer to evaluate against the default 1700-01-01 value like this:

return (new Date(null).toISOString() == dayjs.utc(dateRange1.formattedStartValue).toISOString());

**Comment by raheeliftikhar5** [comment]
@agaitan026 Thank you for the patience. This is fixed now and will be released in next version 2.4.1
